productPage/productPage.css

```diff
@@ -0,0 +1,242 @@
+@import url('https://fonts.googleapis.com/css2?family=Karla:wght@200;300;400;500;600;700;800&family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+/* * {
+    font-family: 'Montserrat', 'sans-serif';
+} */
+
+.grid-container {
+    display: grid;
+    grid-template-columns: (3, 1fr);
+    grid-template-areas:
+        'navbar navbar navbar'
+        'sidebar content content'
+        'sidebar content content'
+        'sidebar content content'
+}
+
+.header {
+    grid-area: navbar;
+    /* border: 2px solid red; */
+}
+
+
+/* Filter Section */
+.filter {
+    /* border: 2px solid red; */
+    grid-area: sidebar;
+    width: 29.4rem;
+    padding: 1.5rem;
+    margin: 15rem auto 5rem auto;
+    
+}
+
+.filters-cta-button {
+    margin-left: 10rem;
+}
+
+.back-btn {
+    width: 10.1rem;
+    height: 4.4rem;
+    left: 4.4rem;
+    top: 21.4rem;
+    border-radius: 1.2rem;
+    border: none;
+    background: #2780F2;
+    font-family: 'Karla';
+    font-style: normal;
+    font-size: 1.8rem;
+    line-height: 2.1rem;
+    color: white;
+    margin: 2rem 0;
+}
+
+.title {
+    /* border: 2px solid red; */
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.title-heading {
+    width: 12.6rem;
+    height: 7.2rem;
+    left: 4.4rem;
+    top: 26.6rem;
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 300;
+    font-size: 4.8rem;
+    line-height: 7.2rem;
+}
+
+.title-left {
+    width: 5rem;
+    height: 1.6rem;
+    left: 26.2rem;
+    top: 29.4rem;
+    font-family: 'Karla';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.3rem;
+    line-height: 1.6rem;
+}
+
+
+.subtitle {
+    font-family: 'Karla';
+    font-style: normal;
+    font-size: 1.8rem;
+    font-weight: 700;
+    line-height: 2.1rem;
+    margin: 1rem 0;
+}
+
+.subtitle-price {
+    /* border: 2px solid magenta; */
+    margin: 1.5rem 0;
+}
+
+.range-slider {
+    width: 100%;
+}
+
+.subtitle-categories {
+    /* border: 2px solid mediumblue; */
+    margin: 1.5rem 0;
+}
+
+article {
+    padding: .2rem 0;
+    font-family: 'Karla';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.4rem;
+    line-height: 1.6rem;
+    display: flex;
+    align-items: center;
+}
+
+.items {
+    font-size: 1.3rem;
+    font-weight: normal;
+    margin-left: 1rem;
+    padding: .3rem 0;
+}
+
+.subtitle-brands {
+    /* border: 2px solid violet; */
+    margin: 1.5rem 0;
+}
+
+.subtitle-ratings {
+    /* border: 2px solid rosybrown; */
+    margin: 1.5rem 0;
+}
+
+
+
+/* Search bar  */
+.search-menu {
+    height: 56px;
+    white-space: nowrap;
+    display: flex;
+    flex-shrink: 0;
+    align-items: center;
+    background-color: var(--search-bg-color);
+    border-radius: 1.6rem;
+    width: 100%;
+    max-width: 62.5rem;
+    padding-left: 20px;
+    margin: 15rem auto 5rem auto;
+
+
+}
+
+.search-bar {
+    height: 55px;
```

Comment:
try to avoid the use of px unit after 10px

Reply:
done~!